src/model/user.py

```
@@ -60,6 +60,17 @@ class User(Base):
     )
 
 
+class UserSummary(Base):
+    __tablename__ = "user"  # Points to the existing user table
+    __table_args__ = {"extend_existing": True}  # Allows multiple mappings to one table
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)